src/database/dao/MainDAO.java

```diff
@@ -0,0 +1,28 @@
+package database.dao;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import database.ConnectionManager;
+
+public class MainDAO {
```

Comment:
Please implement the Closeable interface. This will ask you to implement the `close` method, use it to close the connection.